lib/Runtime/ByteCode/ByteCodeDumper.cpp

```diff
@@ -838,6 +838,7 @@ namespace Js
                 Output::Print(L" [%d] = R%d ",data->SlotIndex, data->Value);
                 break;
             case OpCode::LdLocalSlot:
+            case OpCode::LdParamSlot:
```

Comment:
Thanks.

--Paul

From: aneeshdk [mailto:notifications@github.com]
Sent: Tuesday, March 1, 2016 2:12 PM
To: Microsoft/ChakraCore <ChakraCore@noreply.github.com>
Cc: Paul Leathers <pleath@microsoft.com>
Subject: Re: [ChakraCore] Param scope and body scope as sibling scopes (#384)


In lib/Runtime/ByteCode/ByteCodeDumper.cpp<https://github.com/Microsoft/ChakraCore/pull/384#discussion_r54644905>:

> @@ -838,6 +838,7 @@ namespace Js

>                  Output::Print(L" [%d] = R%d ",data->SlotIndex, data->Value);

>                  break;

>              case OpCode::LdLocalSlot:

> +            case OpCode::LdParamSlot:

Yes. I verified that.

—
Reply to this email directly or view it on GitHub<https://github.com/Microsoft/ChakraCore/pull/384/files#r54644905>.